pages/api/users/update.ts

```diff
@@ -0,0 +1,56 @@
+import { PrismaClient } from "@prisma/client";
+import { NextApiRequest, NextApiResponse } from "next";
+import Joi from "joi";
+
+const prisma = new PrismaClient();
+type userDTO = {
```

Comment:
Same nit here -> UserDTO. Also, if you find yourself needing to reuse types, you can use the `interfaces/` folder.

Reply:
@ethanlee16 Should I make a new file for UserDTO or just put it in index.ts?

Reply:
Good question. Perhaps an export from _one_ of the endpoints might make sense here (the index users route, if you have one?) DTOs are API-specific objects, so it might not make sense to have them exist in the global typedef files.